src/Security/Authentication/OpenIdConnect/src/OpenIdConnectHandler.cs

```diff
@@ -736,18 +736,18 @@ protected override async Task<HandleRequestResult> HandleRemoteAuthenticateAsync
 
                 if (Options.SaveTokens)
                 {
-                    SaveTokens(properties, tokenEndpointResponse ?? authorizationResponse);
+                    SaveTokens(properties!, tokenEndpointResponse ?? authorizationResponse);
                 }
 
                 if (Options.GetClaimsFromUserInfoEndpoint)
                 {
-                    return await GetUserInformationAsync(tokenEndpointResponse ?? authorizationResponse, jwt, user, properties);
+                    return await GetUserInformationAsync(tokenEndpointResponse ?? authorizationResponse, jwt!, user!, properties!);
                 }
                 else
                 {
                     using (var payload = JsonDocument.Parse("{}"))
```

Comment:
What in the world? Why is this parsing an empty object?

Reply:
Usually STJ makes sense. Usually 🐴